cajita/unit_test/tstArray2d.hpp

```diff
@@ -303,7 +303,8 @@ void arrayOpTest()
             for ( long l = 0; l < sub_ghosted_space.extent( 2 ); ++l )
                 EXPECT_EQ( host_subview( i, j, l ), 3.0 * scales[l + 2] + 1.0 );
 
-#ifndef KOKKOS_ENABLE_OPENMPTARGET // FIXME_OPENMPTARGET
+#if !defined( KOKKOS_ENABLE_OPENMPTARGET ) &&                                  \
+    !defined( KOKKOS_ENABLE_SYCL ) // FIXME_OPENMPTARGET // FIXME_SYCL
```

Comment:
Because of multiple valued reductions? Or multidimensional parallelism?

Reply:
Here it's multidimensional reductions

Reply:
No longer disabled